src/storybook/story-test-runner/open-story/testplane-open-story.ts

```diff
@@ -11,10 +11,19 @@ interface HTMLElement {
     innerText: string;
 }
 
-type StorybookWindow = Window &
+export type StorybookWindow = Window &
     typeof globalThis & {
-        __HERMIONE_OPEN_STORY__: (storyId: string, remountOnly: boolean, done: (result: string) => void) => void;
-        __STORYBOOK_ADDONS_CHANNEL__: EventEmitter;
+        __HERMIONE_OPEN_STORY__: (
+            storyId: string,
+            storybookGlobals: Record<string, unknown>,
+            remountOnly: boolean,
+            done: (result: string) => void,
+        ) => void;
+        __TESTPLANE_STORYBOOK_INITIAL_GLOBALS__?: Record<string, unknown>;
+        __STORYBOOK_PREVIEW__?: { storeInitializationPromise?: Promise<void> };
```

Comment:
Storybook preview. Here is `storeInitializationPromise` is stored. We can't emit "updateGlobals" until storybook preview is initialized